music/src/core/compat/global_browser.ts

```diff
@@ -18,6 +18,6 @@
  * limitations under the License.
  */
 
-export const fetch = window.fetch;
+export const fetch = window.fetch.bind(window);
 export const performance = window.performance;
```

Comment:
we don't have to do this for performance? just double checking

Reply:
no, it's a (dumb) JS context thing - since fetch is a method and we export it without binding it, it's messing with the internals of the method (as it's somehow being invoked with a context that's *not* the `window` object, probably because of the way webpack is making a module object)

`performance` and `navigator` are each objects, which have the rest of their methods and properties attached to them, so calling a method on them will have the correct context.

Reply:
* waves hands *

Reply:
ah yeah, duh you're right